src/odemis/driver/lle.py

```diff
@@ -148,16 +148,15 @@ def __init__(self, name, role, port, sources, _serial=None, **kwargs):
         model.Emitter.__init__(self, name, role, **kwargs)
 
         self._shape = ()
-        self._max_power = 0.4 # W (According to doc: ~400mW)
-        self.power = model.FloatContinuous(0., (0., self._max_power), unit="W")
+        # Maximum power taken from the manual file 4487-LUM.54-10009B.SPECTRA.X.pdf
+        self._max_power = (0.231, 0.260, 0.196, 0.295, 0.310, 0.256, 0.62)
```

Comment:
The idea here is that the device might actually have less than 7 sources installed. This depends on the sources. In that case the spectra and power could be shorter. So you cannot directly hardcode it here. 

As a matter of fact, most of the systems using this device only have 4 sources. See https://lumencor.com/products/filters-for-spectra-x-light-engines/ for the power (which is dependent) on the sources. Ideally, the `sources` parameter would also have the max power. We could do that, but we'd have to update the YAML file of every single system. An alternative could be to support the power as 6th value in the spectra, eg `[379.e-9, 384.e-9, 390.e-9, 396.e-9, 401.e-9, 0.225]`, but *also* to have a table with known power for the one listed in the website, and use that as a fallback when there are only 5 values.

Reply:
So instead of hardcoding `max_power` directly, I used `DEFAULT_SOURCES_POWERS` dict with these default values and filled `max_power` with only the existing channels. Is that correct?